### DISCUSSION TITLE
Why is there no text-md utility class?

### DISCUSSION THREAD

**Discussion by LeCoupa** [tailwindlabs/tailwindcss#2067] [Help] [answered]
This is something I regularly think about when I work with Tailwind. 😃

Why is there no `text-md` utility class?

**Comment by adamwathan** [comment, marked as answer]
There is! It's called `text-base` :)

**Comment by redxtech** [comment]
Is there a reason for the inconsistent use of `-md`, `-base`, or a lack of a suffix for the middle/default option? I've noticed that for example shadow and rounded have `.shadow`, `.rounded`, '.shadow-md`, and `.rounded-md`, and text has `text-base`, container has `.container` and `.container-md`, max width has `max-w-md`, font weight has `.font-normal` and `.font-medium`, and there are some others as well. I'm wondering if there's a reason for this that I'm missing.

**Reply by royvanv** [reply]
A class named `text` would not have any semantic meaning. `text-base` indicates it's the base size.
